netdata-installer.sh

```diff
@@ -45,14 +45,14 @@ download() {
 	dest="${2}"
```

Comment:
Any change to `download()` function should be included in following "modules":
* Installer
* Updater
* Kickstart

Great example of this is #5439

Reply:
@paulfantom 	don't you want to fix it in #5460?